repository-data/webfiles/src/main/resources/site/freemarker/publicationsystem/publication.ftl

```diff
@@ -34,8 +34,22 @@
 
 
 <#macro fullContentOfPubliclyAvailablePublication>
-    <div class="content-section">
+    <div class="content-section content-section--highlight">
+
+        <div data-uipath="ps.publication.information-types">
+            <#list publication.informationType as type><#if type?index != 0>, </#if>${type?html}</#list>
+        </div>
+
         <h2 class="doc-title" data-uipath="ps.publication.title">${publication.title?html}</h2>
+
+        <#if parentSeries??>
+        This is part of
```

Comment:
Should this be "This publication is part of"?

Reply:
no, agreed on plain "This is part of" with Martin, Peter, ...

Going forward, this will be a resource in CMS (`hst.setBundle`)